str/trtools/merge_str_prep.py

```diff
@@ -61,28 +59,25 @@ def main(
     for id in list(input_vcf_dict.keys()):
         bcftools_job = b.new_job(name=f'{id} {caller} Files prep')
         bcftools_job.image(BCFTOOLS_IMAGE)
-        bcftools_job.storage('20G')
-        bcftools_job.cpu(8)
+        bcftools_job.cpu(4)
+        bcftools_job.storage('15G')
```

Comment:
Is this going to be enough for the full joint-call, or is this stage still per-sample?

Reply:
still per sample